ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -903,4 +903,4 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
 
         return content;
     }
```

Comment:
Is the tunnel management client already capable of using `http.Agent` ?

Reply:
Could you elaborate on this? Initially without these changes, the `TunnelManagementHttpClient` had the `http.Agent` as an optional parameter. The issue is that was never passed in as part of the web socket connection which is what this PR aims to solve

Reply:
Thanks! that's what I was asking about!